fabric-renderer-registries-v1/src/main/java/net/fabricmc/fabric/api/client/rendereregistry/v1/BlockEntityRendererRegistry.java

```diff
@@ -32,10 +32,12 @@
 	BlockEntityRendererRegistry INSTANCE = new BlockEntityRendererRegistryImpl();
 
 	/**
-	 * Register a BlockEntityRenderer for a BlockEntityType. Can be called clientside before the world is rendered
+	 * Register a BlockEntityRenderer for a BlockEntityType. Can be called clientside before the world is rendered.
 	 *
 	 * @param blockEntityType the {@link BlockEntityType} to register a renderer for
-	 * @param blockEntityRenderer a function that returns a {@link BlockEntityRenderer}
+	 * @param blockEntityRenderer a function that returns a {@link BlockEntityRenderer}. This will be called
+	 *                            when {@link BlockEntityRenderDispatcher} is initialized, or immediately if the class
+	 *                            was already loaded.
```

Comment:
```suggestion
	 * @param blockEntityRenderer a function that returns a {@link BlockEntityRenderer}, called
	 *                            when {@link BlockEntityRenderDispatcher} is initialized or immediately if the dispatcher
	 *                            is already available.
```

Reply:
I'd rather use the specific term "loaded" here since this is actually about class-loading the BlockEntityRenderDispatcher in any way.

Reply:
Updated.